.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Could you change the name of this file to something more descriptive? Now it's not a main workflow, it's just the image workflow :)